list.go

```diff
@@ -271,6 +276,18 @@ NextFile:
 							p.IgnoredGoFiles = append(p.IgnoredGoFiles, fname)
 							continue NextFile
 						}
+						matchesPositive, err := regexp.MatchString("go\\d+\\.\\d+", b)
+						if err != nil {
+							return err
+						}
+						if matchesPositive && b != version {
```

Comment:
Version build tags match the declared build tag version and newer. eg: a file with the `go1.5` tag matches go1.5 and go1.6 and go1.7, etc.